config/prod/prow/jobs/custom/test-infra.yaml

```diff
@@ -81,6 +81,22 @@ presubmits:
           secretName: test-account
 
 periodics:
+- cron: "*/3 * * * *" # Every 3 minutes
```

Comment:
Can we also have a comment for this Prow job here?

Reply:
Added